api.go

```diff
@@ -65,22 +79,48 @@ func RegisterEndpoint(w http.ResponseWriter, r *http.Request) {
 		RenderJSON(w, http.StatusInternalServerError, NewResponse(err.Error()))
 		return
 	}
-	store.Set(a.Endpoint, obj, maxItemTime)
+	store.Set(eKey, obj, maxItemTime)
 	RenderJSON(w, http.StatusOK, NewResponse("cool"))
 }
 
-//GetEndpoint renders registered endpoints.
-func GetEndpoint(w http.ResponseWriter, r *http.Request) {
+func getCacheKeys(endpoint, httpMethod string) string {
```

Comment:
I understand this makes method makes less sense right now, but will be needed to add further object "types" to the store.